client/flags/flags.go

```diff
@@ -157,3 +156,10 @@ func ParseGasSetting(gasStr string) (GasSetting, error) {
 		return GasSetting{false, gas}, nil
 	}
 }
+
+func markFlagRequired(cmd *cobra.Command, name string) {
+	err := cmd.MarkFlagRequired(name)
+	if err != nil {
```

Comment:
This is good :+1: - just a tiny suggestion:

```suggestion
	if err := cmd.MarkFlagRequired(name); err != nil {
```